### DISCUSSION TITLE
Observation and  Survey Player for collection editor

### DISCUSSION THREAD

**Discussion by kiranharidas187** [Sunbird-inQuiry/Community#57] [General Discussions]
@vrayulu @pallakartheekreddy We are planning to enable the Preview feature for observation and survey in the collection editor library. As part of this, we have created an approach doc. Please refer to this and let us know your suggestions.

https://docs.google.com/document/d/18FU8ANf6ul8AfJdaO3jJNvLe9HpqGagQSCOav9mpCk4/edit?usp=sharing

cc-  @ramkumar-pacewisdom @vijiurs

**Comment by pallakartheekreddy** [comment]
@kiranharidas187 above design to enable player only in Editor or is there any plan to use same player in consumption portal as well?

**Reply by kiranharidas187** [reply]
We are planning to use the same player on the consumption side also, once the observation and survey creation  flow is live
